### DISCUSSION TITLE
Not saving my samples

### DISCUSSION THREAD

**Discussion by VasileiosLiaskas** [bangcorrupt/hacktribe#262] [Q&A] [answered]
Hello everybody..
My hacktribe (i had installed it in grey e2) dont save my samples.
I am importing samples ,save them in 500+ manually . And when i turn it off and turn on again my samples arent in these slots of memory.
But  importing all samples by e2sample file is working.
Is it the standar process to import samples? With e2samples and  loading file in every start?

**Comment by enorrmann** [comment, marked as answer]
did you Export All sample after importin them ? 
if you want to preserve imported samples you have to go to menu 28 data utility and then export all sample

**Reply by VasileiosLiaskas** [reply]
A ok. thanks
